src/app/(BottomTabNavigation)/Cases/QRCodeScanner/AddCase/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { router, useLocalSearchParams } from 'expo-router';
+import { Image, Text, TouchableOpacity, View, ScrollView } from 'react-native';
+
+import styles from './styles';
+import { uploadCase } from '../../../../../supabase/queries/cases';
+import { Case } from '../../../../../types/types';
+import { formatDate } from '../../utils';
+
+function AddCase() {
+  const { id, title, imageUrl, date, lawFirm, summary } =
+    useLocalSearchParams() as unknown as Case;
+  const addToCases = async () => {
+    uploadCase(id);
+    router.push('/Cases');
+  };
+  return (
+    <View style={{ height: '100%' }}>
+      <ScrollView
```

Comment:
Ignore for now but this screen should be consistent with the Case Summary screen